feat: add support for fetching pactflow AI entitlement (#129)
SmartBear/smartbear-mcp: docs/products/SmartBear MCP Server/contract-testing-with-pactflow.md

  - Perform advanced queries using selectors to understand compatibility within specific branches or environments
  - Support informed deployment decisions by answering "can I deploy version X of this service to production?"

> ### PactFlow AI Credits

Reviewer: ```suggestion
### PactFlow AI Status
```

> - Purpose: Retrieve the AI feature status for the PactFlow account, including whether AI is enabled, the number of remaining and consumed AI credits, and entitlement or permission issues preventing usage.

Reviewer: ```suggestion
- Purpose: Retrieve the status of AI features for the PactFlow workuspace, including whether AI features are enabled, the number of remaining and consumed AI credits, and user-level permissions issues preventing usage.
```
